code/index.html

```diff
@@ -1,30 +1,248 @@
+          <div><label class="signup-inner__form__check" for="man">Man<input class="signup-inner__form__input"
+                type="radio" id="man" name="gender">
+              <span class="signup-inner__form__check__radio"></span></label></div>
+          <div><label class="signup-inner__form__check" for="other">Other<input class="signup-inner__form__input"
+                type="radio" id="other" name="gender">
+              <span class="signup-inner__form__check__radio"></span></label></div>
+          <fieldset class="signup-inner__form__fieldset" class="sigup__fieldset">
+            <legend class="signup-inner__form__fieldset__legend" class="signup-inner__form__fieldset__legend"><label
+                for="user">Username</label></legend>
+            <input class="signup-inner__form__input" type="text" id="user" name="user" placeholder="You Username"
+              required>
+          </fieldset>
+          <fieldset class="signup-inner__form__fieldset" class="sigup__fieldset">
+            <legend class="signup-inner__form__fieldset__legend" class="signup-inner__form__fieldset__legend"><label
+                for="pwd">Password</label></legend>
+            <input class="signup-inner__form__input" type="password" id="pwd" name="pwd" placeholder="You Password"
+              minlength="4" maxlength="8" required>
+          </fieldset>
+          <label for="option">Where did you hear of us?</label>
+          <select class="signup-inner__form__select" name="option">
+            <option value="google">Google</option>
+            <option value="fried">From a friend</option>
+            <option value="internet">Internet adds</option>
+            <option value="local">Local adds</option>
+            <option value="hairdresser">My hairdresser</option>
+            <option value="other">Other</option>
+          </select>
+          <div><label for="privacy" class="signup-inner__form__check"><input class="signup-inner__form__input"
+                type="checkbox" id="privacy" name="privacy" required>
+              <span class="signup-inner__form__check__box"></span>
+              I agree to the <a href="">privacy policy</a></label></div>
+          <div><label for="subscribe" class="signup-inner__form__check"><input class="signup-inner__form__input"
+                type="checkbox" id="subscribe" name="subscribe" required>
+              <span class="signup-inner__form__check__box"></span>Subscribe to the newsletter </label>
+          </div>
+          <input class="signup-inner__form__input" type="submit" value="Sign Up">
+        </form>
+        <p>Already a member? Login <a href="login" target="blank">here</a></p>
       </div>
+      <!-- Image with responsive sizes next to (desktop/tablet) or on top of (mobile) form-->
+      <picture class="signup__image">
+        <source media="(max-width: 600px) and (max-device-width: 600px)" srcset="./images/colors350px.jpg">
```

Comment:
We don't have to specify both `max-width` and `max-device-width` - the first one is completely enough :)